src/Analyzers/AnalyzerRunner.cs

```diff
@@ -45,7 +39,7 @@ public AnalyzerRunner(bool includeCompilerDiagnostics)
 
             // If are not running any analyzers and are not reporting compiler diagnostics, then there is
             // nothing to report.
-            if (analyzers.IsEmpty && !_includeComplilerDiagnostics)
+            if (analyzers.IsEmpty && !includeCompilerDiagnostics)
```

Comment:
it feels like we could save on the boilerplate of passing this down if we just compute 'includeCompilerDiagnostics' here from the list of analyzers.

Reply:
perhaps there are tests which need to directly specify this?

Reply:
it is calculated from the list of codefixproviders.

Reply:
If a test is testing this method directly, they could pass this value in as desired. If they were testing this indirectly, they would need to build and add an appropriate AnalyzerReference containing a fixer for a compiler diagnostic.

Reply:
Oops, my mistake!